content/2019/form-decoding.md

```diff
@@ -163,41 +115,27 @@ type Error
     | AgeNegative
     | AgeRequired
     ...
-    ...
 
 toGoat2 : RegisterForm -> Result (List Error) Goat
 ```
 
-If user inputs are invalid, it returns `Err` like:
+For which a possible return value could be:
 
 ```elm
 Err [ NameRequired, AgeInvalidInt ]
 ```
 
-And it only returns converted value with `Ok` keyword:
-
-```elm
-Ok (Goat "Sakura-chan" 2 ......)
-```
-
-It's obvious that it is also doing form validation. I confidently say that form decoding is the new-generation of form validation for statically typed programming!
```

Comment:
Sure, I'll put that in!

I know a few people who tend to avoid using "obvious" in that way because it can imply that if the reader didn't see whatever is obvious they must be less intelligent. This isn't a universal implication. Some people wouldn't read it that way at all! But some would. (Oh the joys of languages and cultural connotations.) I tend to lean toward using "clear", as in "It's clear that..."

Reply:
Nice! I'll use "clear" rather than "obvious" in such way.